src/main/java/com/example/earthtalk/domain/news/service/CrawlConfig.java

```diff
@@ -0,0 +1,23 @@
+package com.example.earthtalk.domain.news.service;
+
+import com.example.earthtalk.domain.news.entity.NewsSite;
+import com.example.earthtalk.domain.news.entity.NewsType;
+import com.example.earthtalk.global.constant.ContinentType;
+import java.util.List;
+import java.util.Map;
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.stereotype.Component;
+
+@Configuration
+@ConfigurationProperties("news")
+@Setter
+@Getter
+public class CrawlConfig {
+
+    private List<NewsSite> sites;
```

Comment:
setter 대신에 생성자로 바인딩하는 것은 어떨까요?? final 변수로 선언해서 안정성을 확보하기 위해서요!
```suggestion
@Configuration
@Getter
@ConfigurationProperties("news")
@RequiredArgsConstructor
public class CrawlConfig {

    private final List<NewsSite> sites;
```

Reply:
저렇게 변경하니까 
Annotated with @ConstructorBinding but defined as Spring component
이런 컴파일 에러가 뜨네요?-?

Reply:
전부다 수정해서 푸쉬했습니다!